python/lsst/meas/algorithms/objectSizeStarSelector.py

```diff
@@ -49,6 +53,23 @@ class ObjectSizeStarSelectorConfig(BaseSourceSelectorTask.ConfigClass):
         default=0.0,
         check=lambda x: x >= 0.0,
     )
+    doSignalToNoiseLimit = pexConfig.Field(
+        doc="Apply signal-to-noise (i.e. flux/fluxErr) limit to Psf Candidate selection?",
+        dtype=bool,
+        default=False,
+    )
+    signalToNoiseMin = pexConfig.Field(
+        doc="specify the minimum signal-to-noise for good Psf Candidates",
+        dtype=float,
+        default=20.0,
+        check=lambda x: x >= 0.0,
```

Comment:
Is there are reason that this class should never work with negative flux objects? This is related to the fluxMin/Max fields. I understand that this class is mainly used for selection PSF fitting objects, but there is nothing necessarily stopping this from being used, for instance on catalogs produced by difference images.

Reply:
This is technically a preexisting condition (well, slightly derived from the preexisting fluxMin condition!).  I'm not in the market for making any such changes on this ticket, but I do think it's true that this selector is only being used in PSF modeling candidate selection (I'm not sure what the diffim selectors do...)

Reply:
That's fair, it may be worth ticketing this for the future and converting these checks into a `validate` method that could, for instance, check that the min value is less that the max value.